src/metabase/models/pulse.clj

```diff
@@ -259,7 +259,7 @@
   "Fetch an Alert or Pulse, and do the 'standard' hydrations, adding `:channels` with `:recipients`, `:creator`, and
   `:cards`."
   [notification-or-id & additional-conditions]
-  (some-> (apply Pulse :id (u/the-id notification-or-id), additional-conditions)
+  (some-> (apply db/select-one Pulse :id (u/the-id notification-or-id), additional-conditions)
```

Comment:
We were supposed to be linting against this earlier, but I think earlier Kondos were thrown off by the indirection with `apply`/`map`. Now it's being caught.